.env.test

```diff
@@ -45,3 +45,9 @@ ATLASSIAN_URL=https://test-atlassian-instance.atlassian.net
 # Use `trace` to get verbose logging or `info` to show less
 LOG_LEVEL=info
 WEBHOOK_PROXY_URL=http://localhost:8080/github/events
+
+# Cryptor
+CRYPTOR_SIDECAR_BASE_URL=http://127.0.0.1:26272
+CRYPTOR_SIDECAR_CLIENT_IDENTIFICATION_CHALLENGE="GitHub for Jira client"
```

Comment:
What's this?

Reply:
I think it is related to this [bit](https://developer.atlassian.com/platform/cryptor/integration/integrating-sidecar/#enabling-ssrf-protection)